src/api/scanned-codes.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+parent: API Reference
+title: Scanned Codes
+permalink: /api/scanned-codes
+---
+
+# Scanned Codes
+
+A scanned code is a barcode that a merchant scans. The code can be used to create a [Payment Request][].
+
+## Contents
+{:.no_toc .text-delta}
+
+* TOC
+{:toc}
+
+## Models
+
+### Scanned Code
+
+{% h4 Mandatory Fields %}
+
+|      Field       |  Type  |                                              Description                                               |
+| :--------------- | :----- | :----------------------------------------------------------------------------------------------------- |
+| code             | String | The utf8 representation of data decoded from what was scanned.                                         |
+| scannedBy        | String | The person that scanned the code. Can be `merchant`.                                                   |
+| provider         | String | The integrator that owns the code. The provider can be used to determine what [Line Items][] to apply. |
```

Comment:
Can we have a list that you can get back from this?
```suggestion
| provider         | String | The integrator that owns the code. This can be used to calculate discounts on [Line Items][]. Can be `paypal`, `venmo`, `farmlands` or `centrapay`. |
```